blockscout-ens/bens-logic/src/subgraphs_reader/sql/domain.rs

```diff
@@ -18,12 +19,22 @@ resolved_address,
 resolver,
 to_timestamp(ttl) as ttl,
 is_migrated,
-to_timestamp(created_at) as created_at,
+to_timestamp(created_at) as registration_date,
 owner,
 registrant,
 wrapped_owner,
 to_timestamp(expiry_date) as expiry_date,
-COALESCE(to_timestamp(expiry_date) < now(), false) AS is_expired 
+COALESCE(to_timestamp(expiry_date) < now(), false) AS is_expired
+"#;
+
+const DOMAIN_DEFAULT_SELECT_CLAUSE: &str = r#"
+id,
+name,
+resolved_address,
+to_timestamp(created_at) as registration_date,
+owner,
+to_timestamp(expiry_date) as expiry_date,
+COALESCE(to_timestamp(expiry_date) < now(), false) AS is_expired
```

Comment:
Since I need to cast created_at, expiry_date to timestamp, I need to write all fields (asteriks `*` wont work here)
From other since, I cannot re use DETAILED_DOMAIN select clause, because it contains other_addresses field. Of course I can create 3rd variable, but this is overcomplication for nothing.
Also, on server side we don't need to include all other fields from domain. however If someday we will need them, I will change code to include all domain fields